Connectors/Revit/Speckle.Connectors.RevitShared/Bindings/RevitSendBinding.cs

```diff
@@ -103,11 +114,92 @@ fidelityString is not null
         }
       }
       _detailLevelCache[modelCard.ModelCardId.NotNull()] = fidelity;
-      return new ToSpeckleSettings(fidelity);
+      return fidelity;
     }
+
     throw new ArgumentException($"Invalid geometry fidelity value: {fidelityString}");
   }
 
+  private Transform? GetReferencePointSetting(SenderModelCard modelCard)
```

Comment:
A couple of question/comment around this and change requests accordingly;

1. Do these 3 types of reference points always generate same transforms in model same or not? If so why we calculate them always? I would cache them.

2. `SendBinding` started to bubble up, nice to have if we extract "GetSendSettings" to outside

Reply:
> Do these 3 types of reference points always generate same transforms in model same or not? If so why we calculate them always? I would cache them.

Unfortunately we aren't tracking if the reference points in the doc are changed between send operations, so for now the transform needs to be recalculated every time. This should be *ok* performance-wise bc it should only happen once per send operation.